src/_data/sites/hi.yml

```diff
@@ -0,0 +1,521 @@
+          योगदान में कोड सबमिट करना, दस्तावेज़ लिखना, डिस्कॉर्ड सर्वर पर सवालों के जवाब देना, और बहुत कुछ शामिल हैं। यह पैसा टीएससी द्वारा मासिक रूप से वितरित किया जाएगा।
+      - name: निर्भर परियोजनाएं
+        text: >
+          हम अपनी निर्भरता का समर्थन करने में सहायता के लिए प्रत्येक माह DEPENDENCIES_BUDGET USD अलग रखते हैं।
+          हमारा मानना है कि ओपन-सोर्स प्रोजेक्ट्स, जो वित्त पोषित होने के लिए भाग्यशाली हैं, उन्हें छोटी परियोजनाओं का समर्थन करने में मदद करने की ज़िम्मेदारी है जिन पर वे निर्भर हैं।
+      - name: सामुदायिक परियोजना
+        text: >
+          हम प्लगइन्स, पार्सर्स और अन्य परियोजनाओं के लिए हर महीने COMMUNITY_BUDGET USD सुरक्षित करते हैं जो पूरे ESLint पारिस्थितिकी तंत्र के लिए फायदेमंद हैं।
+          हम मानते हैं कि परियोजना की ताकत इसके पारिस्थितिकी तंत्र में निहित है, हमें उम्मीद है कि महत्वपूर्ण परियोजनाओं को बनाए रखा जाएगा।
+      - name: समर्थन प्रणाली
+        text: >
+          हम हर महीने एक छोटी राशि का उपयोग उस सॉफ़्टवेयर के भुगतान के लिए करते हैं जिसका उपयोग टीम प्रबंधन में सहायता के लिए करती है जिसमें Google Workspace और cloud storage जैसी चीज़ें शामिल हैं।
+    footer:
+      text: >
+        इसके अतिरिक्त, समय-समय पर हम ठेकेदारों को परियोजना के लिए महत्वपूर्ण चीजों पर काम करने के लिए भुगतान करने के लिए धन का उपयोग करते हैं
+        जहां टीम के पास काम को खुद पूरा करने के लिए समय या विशेषज्ञता नहीं है।
+        उदाहरण के लिए, हमने हाल ही में एक डिज़ाइनर और डेवलपर को इस वेबसाइट को फिर से डिजाइन करने के लिए काम पर रखा है।
+        हम यह सुनिश्चित करते हैं कि हम हर महीने जितना धन अर्जित करते हैं उससे कम खर्च करें ताकि हम ऐसी परियोजनाओं के लिए बचत कर सकें।
+  donation_tiers:
+    title: दान स्तरीय प्रणाली
+    subtitle: हालाँकि हम किसी भी आकार का दान स्वीकार करते हैं, हमारे पास प्रत्येक स्तर पर अलग-अलग पुरस्कारों के साथ एक स्तरीय प्रणाली है।
+    cta:
+      open_collective_text: >
+        <span class="visually-hidden">Open Collective</span> पर दान करें
+      github_sponsors_text: >
+        <span class="visually-hidden">GitHub</span> पर दान करें
+    items:
+      - name: Platinum Sponsor
+        amount: $2000+
+        frequency: प्रति माह
+        text: 2 घंटे का समर्थन, होमपेज और README के शीर्ष पर लोगो पोस्ट किया गया। इसके अलावा, यह Gold Tier की सभी सामग्री प्रदान करता है।
+      - name: Gold Sponsor
+        amount: $1000–1999
+        frequency: प्रति माह
+        text: लोगो को शीर्ष पृष्ठ पर और README की दूसरी पंक्ति पर पोस्ट किया जाएगा, और हमारे (ESLint) ट्विटर अकाउंट (20,000 फॉलोअर्स) से धन्यवाद ट्वीट और धन्यवाद ब्लॉग लेख पोस्ट किए जाएंगे।
+      - name: Silver Sponsor
+        amount: $500–999
+        frequency: प्रति माह
+        text: लोगो को शीर्ष पृष्ठ और README की तीसरी पंक्ति पर पोस्ट किया जाएगा।
+      - name: Bronze Sponsor
+        amount: $200–499
+        frequency: प्रति माह
+        text: लोगो को शीर्ष पृष्ठ और README की चौथी पंक्ति पर पोस्ट किया जाएगा।
+  testimonials:
+    title: testimonials
```

Comment:
Let's replace it with `प्रशंसापत्र` or something. WDYT?

Reply:
Yep, we can do that.